draft-private-access-tokens.md

```diff
@@ -727,9 +727,42 @@ signal, such as an IP address, especially since malicious clients can generate g
 Private Access Tokens and for Origins to work. However, similar DoS vectors already exist
 for Origins, e.g., at the underlying TLS layer.
 
+## Channel Security
+
+An attacker that can act as an intermediate between Mediator and Issuer communication can
+influence or disrupt the ability for the Issuer to correctly rate-limit token issuance.
+All communication channels MUST use server-authenticated HTTPS. Where appropriate, e.g., between
+Clients and Mediators, connections MAY mutually authenticate both client and server, or use mechanisms
+such as TLS certificate pinning, to mitigate the risk of channel compromise.
+
+An attacker than can intermediate the channel between Client and Origin can
+observe a TokenChallenge, and can view a Token being presented for authentication
+to an Origin. Origins can avoid this Token being used for some future connection
+by the attacker impersonating the Client by ensuring that the redemption_nonce
+presented in the TokenChallenge is bound to the specific TLS session with the
+Client.
 
 # Privacy Considerations {#privacy-considerations}
 
+## Origin Verification
+
+Private Access Tokens are defined in terms of a Client authenticating to an Origin, where
+the "origin" is used as defined in {{?RFC6454}}. In order to limit cross-origin correlation,
+Clients MUST verify that the origin_name presented in the TokenChallenge structure ({{scheme}})
+matches the origin that is providing the HTTP authentication challenge, where the matching logic
+is defined for same-origin policies in {{?RFC6454}}. Clients MAY further limit which
+authentication challenges they are willing to respond to, for example by only accepting
+challenges when the origin is a web site to which the user navigated.
+
+## Client Identification with Unique Keys
+
+Client activity could be linked if an Origin and Issuer collude to have unique keys targeted
+at specific Clients or sets of Clients. In order to mitigate this risk, the Mediator can 
+observe and validate the key_id presented by the Client to the Issuer. As described in
+{{issuance}}, Mediators MUST validate that the key_id in the Client's AccessTokenRequest
+matches a known public key for the Issuer. This validation allows for key rotation across
```

Comment:
Probably as a separate issue this probably needs to be more nuanced. The issuer could decide to do a key rotation when a specific user/event happens (tracking the first 100 users vs the next set of user).

Mediators may almost need their own key rotation policy (or they delegate that to whatever known key service they use).

Reply:
Modified the text a bit on Jana's suggestion, but yes, we can continue to improve it.